components/Buttons/MainButton/MainButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Link from "next/link";
+
+import ButtonBase from "@material-ui/core/ButtonBase";
+import Typography from "@material-ui/core/Typography";
+
+import { useStyles } from "./styles";
+
+type props = {
+  url: string;
+  title: string;
+  href: string;
+};
+
+export default function MainButton({ title, url, href }: props) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Link href={href}>
+        <ButtonBase
+          focusRipple
+          key={title}
+          className={classes.image}
+          focusVisibleClassName={classes.focusVisible}
+          style={{
+            width: "80vw",
+          }}
```

Comment:
Extract to class. In such a way, the style object will be recreated on every render.

Reply:
![image](https://user-images.githubusercontent.com/47990852/123702739-cc8bad80-d863-11eb-923c-9a8975b2024e.png)
it's example from material-ui itself. Or am I wrong?

https://v1.material-ui.com/demos/buttons/

Reply:
In the example, you've got dynamic value from an element, really. You're hardcoding text "80vh".